app/helpers/posts_helper.rb

```diff
@@ -28,9 +28,14 @@ def max_body_length(_category)
   end
 
   # @param category [Category, Nil]
+  # @param post_type [PostType]
   # @return [Integer] the minimum length for post titles
-  def min_title_length(category)
-    category&.min_title_length || 15
+  def min_title_length(category, post_type)
+    if ['HelpDoc', 'PolicyDoc'].include?(post_type.name)
```

Comment:
suggestion: use your new `PostType#system?` method?

Reply:
Yeah, good idea, forgot I even added it!